generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>“## </p><p>Sampling of a new segmentation mask for an input image of the validation set”</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>